components/PropRow.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+import styles from '../style/base';
+
+export default function PropRow({ left, right }) {
+  return (
+    <View style={styles.propRow}>
```

Comment:
usually it's better to keep `styles` used by the component next to the component and extract only shared properties (like font, colors etc.) as otherwise your `base.tsx` file may grow pretty fast and become hard to maintain.

Also, there's probably very small chance that you will reuse `styles.propRow` or `styles.properties` somewhere else -> you will probably use `PropRow` then or create new styles 😄